test/queue.test.ts

```diff
@@ -46,7 +46,8 @@ describe('Test Queue class', function () {
         expires: undefined,
         deadLetterExchange: undefined,
         deadLetterRoutingKey: undefined,
-        maxLength: undefined
+        maxLength: undefined,
+        arguments: {}
```

Comment:
❓ Were you having issues with arguments being undefined?

Reply:
because of destructed  arguments  like that
`arguments: {...this.option.arguments}`  in case of the undefined test cases are breaking.